### ISSUE TITLE
Elevated circulating 1-methylhistidine concentration

### ISSUE THREAD

**Issue by JillMiller** [obophenotype/human-phenotype-ontology#6432]
**Preferred term label:** Elevated circulating 1-methylhistidine concentration

**Synonyms**

**Definition (free text, please give PubMed ID)** Concentration of 1-methylhistidine in the blood is above the normal range. PMID: 32649643; PMID: 32235743

**Parent term (use hpo.jax.org/app)** Hyperhistidinemia HP:0010906

**Diseases characterized by this term ? (e.g. Orphanet or OMIM number)**

**Your nano-attribution (ORCID)**

**Comment by pnrobinson** [comment]
added term
